keps/sig-node/4742-node-topology-downward-api/kep.yaml

```diff
@@ -0,0 +1,44 @@
+title: KEP Template
+kep-number: 4742
+authors:
+  - "@andrewsykim"
+  - "@docandrew"
+owning-sig: sig-node
+participating-sigs:
```

Comment:
since we are changing this KEP from kubelet doing something to scheduler copies all the labels, I suggest to at a minimum add a sig-scheduling here or even better completely move it over to the sig-scheduling.

SIG Node may have input on number of labels and information exposure, but ultimately it should be sig scheduling conversation

Reply:
It's not really the scheduler here, it's all in the API server.  But specifically it's in the Pod strategy code mostly.

Reply:
I'm inclined to leave it in sig-node since most of the considerations here are around how the node topology labels are consumed via downward API.

Reply:
good with me. Since I don't have much knowledge I asked a few questions on that part of the house like whether suggested copying happens before or after customers' mutating webhooks. Sorry if those are trivial questions